komga-webui/src/locales/en.json

```diff
@@ -203,12 +202,12 @@
     "nothing_to_show": "Nothing to show",
     "outdated": "Outdated",
     "pages": "pages",
-    "pages_n": "No pages | 1 page | {count} pages",
+    "pages_n": "No pages | One page | {count} pages",
     "password": "Password",
-    "pending_tasks": "No pending tasks | 1 pending task | {count} pending tasks",
+    "pending_tasks": "No pending tasks | One pending task | {count} pending tasks",
```

Comment:
the figure notation is important for display size, "One" in other languages could be very long, and overflow.

Reply:
`max-length:10`? https://docs.weblate.org/en/latest/user/checks.html?highlight=flag#maximum-length-of-translation

Reply:
see above